app/controllers/map_proxy_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class MapProxyController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def proxy
+    send_response(client.proxy)
```

Comment:
I feel like at this point we could just inline `send_response` rather than having a separate method.